Orange/widgets/visualize/owmosaic.py

```diff
@@ -383,19 +382,34 @@ def init_combos(self, data):
             if attr.is_discrete or attr.is_continuous:
                 for combo in self.attr_combos:
                     combo.addItem(icons[attr], attr.name)
+        self.cb_attr_color.addItem("(Pearson residuals)")
+        for attr in chain(data.domain.class_vars, data.domain.attributes, data.domain.metas):
```

Comment:
Yes, DomainModel enables us to add a custom string at the beginning. Now class variables, attributes and metas are added into a combo in an alphabetical order.